docs/COMPILING.md

```diff
@@ -8,21 +8,23 @@
 1. Run `git clone https://github.com/FunkinCrew/funkin.git` to clone the base repository.
```

Comment:
Very goob, I just think that it should be added that you need to `cd` to the funkin directory before performing step two, or else you get issues like this XD
* #2973